modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/pipeline_constructs/deploy_pipeline_construct.py

```diff
@@ -127,6 +131,12 @@ def __init__(
                     "PREPROD_REGION": codebuild.BuildEnvironmentVariable(value=preprod_region),
                     "PROD_ACCOUNT": codebuild.BuildEnvironmentVariable(value=prod_account),
                     "PROD_REGION": codebuild.BuildEnvironmentVariable(value=prod_region),
+                    "PRE_PROD_VPC_ID": codebuild.BuildEnvironmentVariable(value=preprod_vpc_id),
```

Comment:
Just noticed this - the properties are passed to deploy CodeBuild but we also need to use them in the [deploy_app](https://github.com/awslabs/mlops-modules/tree/main/modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/seed_code/deploy_app) (use [constants](https://github.com/awslabs/mlops-modules/blob/main/modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/seed_code/deploy_app/config/constants.py) to read them and pass in [app.py](https://github.com/awslabs/mlops-modules/blob/main/modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/seed_code/deploy_app/app.py#L35)) to pass [VPC config](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_sagemaker.CfnEndpointConfig.html#vpcconfig-1) to the [endpoint](https://github.com/awslabs/mlops-modules/blob/main/modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/seed_code/deploy_app/deploy_endpoint/deploy_endpoint_stack.py#L172).

Let's catch up I will guide you how to add them.